models/app.model.js

```diff
@@ -37,4 +37,22 @@ exports.fetchCommentsByArticle=({article_id})=>{
                 WHERE article_id=$1
                 ORDER BY created_at DESC`,[article_id])
         }).then(({rows})=>rows)
+}
+
+exports.addCommentToDatabase=({username,body},{article_id})=>{
+    return exports.fetchArticleById({article_id})
+        .then(()=>{
+            return db.query(`INSERT INTO comments
+                (votes,author,body,article_id)
+                VALUES
+                ($1,$2,$3,$4)
+                RETURNING *`,[
+                    0,
+                    username,
+                    body,
+                    article_id
+                ])
```

Comment:
This is so minor but I'm struggling to find things to criticise 😂  you don't need to pass 0 for votes because the database will automatically assign it 0